rollup.config.js

```diff
@@ -0,0 +1,93 @@
+/* eslint-env node */
+'use strict';
+
+const pkg = require('./package.json');
+const commonjs = require('rollup-plugin-commonjs');
+const resolve = require('rollup-plugin-node-resolve');
+const json = require('rollup-plugin-json');
+const babel = require('rollup-plugin-babel');
+const { terser } = require('rollup-plugin-terser');
+
+const DEST_DIR = 'dist';
+const GLOBAL_LIBRARY_NAME = 'purecloud.apps.ClientApp';
+const CJS_FILENAME = 'main.js';
+const ES_FILENAME = 'main.mjs';
+const BROWSER_FILENAME = `${pkg.name}.js`;
+const PROD_BROWSER_EXT = '.min.js';
+
+const LEGAL_TEXT = `/*
+* ${pkg.name}
+* @copyright Copyright (C) ${new Date().getUTCFullYear()} Genesys Telecommunications Laboratories, Inc.}
+* @license MIT
+*
+* This software comprises other FOSS software.
+* Attribution and license information can be found at https://github.com/MyPureCloud/client-app-sdk/blob/master/README.md
+*/`;
+
+const baseRollupConfig = {
+    input: './src/index.js',
+    cache: false,
+    plugins: [
+        commonjs(),
+        resolve(),
+        json(),
+        babel({
```

Comment:
I know this is copied from just what I had, but I'm trying to remember how the babel config worked way back when.  Not sure if it was pulling in the babel plugins automatically and if we're still getting them.  Looks like we have a few polyfills in there like object assign and external helpers.  Also, it references preset-latest which is deprecated.

For this lib, I figure we keep it pretty unopinionated and mostly let rollup deal with the modules and keep it pretty vanilla.  But, for the umd bundle for the cdn, maybe we need different babel config using presets?

Also, we can punt on this if OBE'd by follow on typescript work.

Let's chat a bit tomorrow about this.

Reply:
The diffs from current master and this branch are pretty close though, so, I think we're close.

Reply:
This config will automatically pull in the presets/plugins from the `.babelrc` in the `src` folder and there shouldn't be any changes to the babel config in this PR. However, [in this PR](https://github.com/MyPureCloud/client-app-sdk/pull/71/files), I went ahead and upgraded babel for us and dropped preset-latest in favor of `@babel/preset-env` along with some other config changes. And then in the typescript PR there were also a few tweaks I had to make to the babel config. I'm happy to talk through this today!